pkg/brokerapi/service_binding.go

```diff
@@ -44,9 +44,4 @@ type CreateServiceBindingResponse struct {
 
 // Credential represents connection details, username, and password that are
 // provisioned when a consumer binds to a service instance
-type Credential struct {
-	Hostname string `json:"hostname"`
-	Port     string `json:"port"`
-	Username string `json:"username"`
-	Password string `json:"password"`
-}
+type Credential map[string]interface{}
```

Comment:
I can make it a string,it just means moving the Marshal() from down below to here.